src/file_pipe_log/log_file.rs

```diff
@@ -30,19 +30,14 @@ const FILE_ALLOCATE_SIZE: usize = 2 * 1024 * 1024;
 pub struct LogFd(RawFd);
 
 fn from_nix_error(e: nix::Error, custom: &'static str) -> std::io::Error {
-    match e {
-        nix::Error::Sys(no) => {
-            let kind = std::io::Error::from(no).kind();
-            std::io::Error::new(kind, custom)
-        }
-        e => std::io::Error::new(std::io::ErrorKind::Other, format!("{}: {:?}", custom, e)),
-    }
+    let kind = std::io::Error::from(e).kind();
+    std::io::Error::new(kind, custom)
```

Comment:
Handle the case where kind == Other?

Reply:
In nix 0.23, all error types have been unified to Errno, and it appears that the `Other` kind no longer needs to be handled

Reply:
I see.